modin/pandas/base.py

```diff
@@ -11,7 +11,37 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Implement DataFrame/Series public API as pandas does."""
+"""
+Implement DataFrame/Series public API as pandas does.
+
+Methods docstrings contain tables with supported parameters information.
+These tables are structed as follows: The first column contains the parameter name,
+the second, third and fourth column - flags describing particular properties of
+method parameters for a concrete execution supported by Modin (`PandasOnRay`, `PandasOnDask`
+or `OmniSci`), and the last one is for parameter notes. The flags stand for the following:
+
++-----------+-----------------------------------------------------------------------------------------------+
+| Flag      | Meaning                                                                                       |
++===========+===============================================================================================+
+| Harmful   | Usage of this parameter can be harmful for performance of your application. Usually this      |
+|           | happens when parameter (full range of values and all types) is not supported and default      |
+|           | pandas implementation is used                                                                 |
++-----------+-----------------------------------------------------------------------------------------------+
+| Non-lazy  | Usage of this parameter can trigger non-lazy execution (actual for OmniSci execution only)    |
++-----------+-----------------------------------------------------------------------------------------------+
+| Partial   | Parameter can be partly unsupported, it's usage can be harmful for performance of your        |
+|           | appcication. This can happen if some parameter values or types are not supported (for example |
+|           | boolean values are suported while integer are not) and default pandas implementation is used  |
++-----------+-----------------------------------------------------------------------------------------------+
+| pandas    | Usage of this parameter, triggers usage of original pandas function as is, no performance     |
+|           | degradation/improvement should be observed                                                    |
++-----------+-----------------------------------------------------------------------------------------------+
```

Comment:
I don't see this flag anywhere. In addition, even if we use pandas functions, we still have a slight slowdown in converting a frame to a pandas type.

Reply:
Now it's clear, then it will need to be added later.